static/js/around-map.js

```diff
@@ -107,3 +109,19 @@ function urlFor (id) {
 function findPoi (id) {
   return pois.find(poi => poi._id == id)
 }
+
+function highlight(poi_id) {
```

Comment:
Please be consistent with spaces for functions. I don't have a preference whether or not to put it, but just keep the same way of writing it all the way for all function in this project:
`function myFuncName()` or `function myFuncName ()`.